lib/kits/core-ui/components/helpers/translate.js

```diff
@@ -0,0 +1,38 @@
+import * as locales from '../../../../../locales';
+
+const getTranslationOverride = (dataset = {}) => {
+    const keys = Object.keys(dataset);
+    const translationsKeyVal = {};
+    keys.filter((key) => key.includes('translationKeys-')).map((key) => {
```

Comment:
use `forEach` instead of `map` if you're only looping for side-effects and not using the new array that `map` returns.